Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -202,25 +175,31 @@ def createCodeReviewExec(String jobcard, String ruleFile, String customRuleFile,
 	// add identified syslib
 	syslib.each { jcl+="//         DD DISP=SHR,DSN=${it} \n"}
 	// add syslib libraries from property file
-	if (props.codereview_syslib){
-		props.codereview_syslib.split(',').each { jcl+="//         DD DISP=SHR,DSN=${it} \n" }
+	if (props.codereview_syslib) {
+		props.codereview_syslib.split(',').each {
+			jcl+="//         DD DISP=SHR,DSN=${it} \n"
+		}
 	}
 
-	if ( PropertyGroupFile ) {
+	if (PropertyGroupFile) {
```

Comment:
Let's use lowercase names
```suggestion
	if (propertyGroupFile) {
```

Reply:
This needs to be applied on the method  and below as well . Unfortunately, I am not allowed to mage this suggestion